python/testing/test_camera.py

```diff
@@ -104,14 +104,33 @@ def test_state_compare():
 def test_moves():
     engine = f3d.Engine.create(True)
     camera = engine.window.camera
+    pos = -1, 0, 0
+    foc = 0, 0, 0
+    up = 0, 1, 0
 
-    camera.dolly(10)
+    camera.position = pos
+    camera.focal_point = foc
+    camera.view_up = up
     angle = 30
-    camera.roll(angle)
+
     camera.azimuth(angle)
+    az_value = camera.get_azimuth()
+    assert math.isclose(
+        az_value, angle, abs_tol=1e-3
+    ), f"Expected azimuth ~{angle}, got {az_value}"
```

Comment:
```suggestion
    assert az_value == approx(angle)
```
using `from pytest import approx` is less noisy than explicitly calling `isclose`. 

Also there's no need for an handcrafted description because `pytest` already does it on failure, for example:
```
>       assert az_value - 1 == approx(angle)
E       assert 28.999999999999996 == 30 ± 3.0e-05
E         
E         comparison failed
E         Obtained: 28.999999999999996
E         Expected: 30 ± 3.0e-05
```

Reply:
Okay 👍.